README.md

```diff
@@ -17,7 +17,7 @@ $ composer require skrz/bunny-bundle
 Then add `BunnyBundle` to Symfony Kernel:
 
 ```php
-use Skrz\Bundle\BunnyBundle\SkrzBunnyBundle;
+use Skrz\Bunny\Exception\SkrzBunnyBundle;
```

Comment:
Nemá to být jen `Skrz\Bunny\SkrzBunnyBundle`?

Reply:
readme jsem vubec neupravoval mam to v planu upravit az budeme migrovat prvni projekt